### DISCUSSION TITLE
Speed up inference time.

### DISCUSSION THREAD

**Discussion by danielsoy** [ultralytics/yolov5#5208] [Q&A] [answered]
Hi ppl.
 I have already deployed my yolov5s model running in an intel I3 core 9100F with
a nvidia GT210  gpu, that is not cuda capable.

Here u can see that the inference time, is around 200 ms.
[https://www.youtube.com/watch?v=hUnlAcAdKB8&ab_channel=tektronix475]

What GPU could i choose that runs faster than my 9100 CPU, around U$ 150 price range?

**Comment by glenn-jocher** [comment, marked as answer]
@danielsoy I think almost any usable GPU will run faster than CPU, i.e. an NVIDIA 1080 Ti would be an easy place to start if you are on a budget.

Even the K80, which is Colab's slowest GPU from several years ago runs default inference in about 30 ms:

<img width="810" alt="Screen Shot 2021-10-15 at 11 22 10 AM" src="https://user-images.githubusercontent.com/26833433/137534698-ede95a3f-e59a-447f-be44-e8a73028fea7.png">

**Reply by danielsoy** [reply]
I can see that the 1080 Ti, has a compute capability of  6.1.

So i will try to get something affordable for me, in this range.

The python inference task, is taking 95% of the CPU bandwidth btw...

Thanks again Glenn!
